readthedocs/projects/tasks.py

```diff
@@ -1351,7 +1384,7 @@ def _manage_imported_files(version, path, commit):
                 created_html_files.append(obj)
 
     # Send bulk_post_create signal for bulk indexing to Elasticsearch
-    bulk_post_create.send(sender=HTMLFile, instance_list=created_html_files)
+    bulk_post_create.send(sender=PageDocument, instance_list=created_html_files)
```

Comment:
Whats the reason to change the `sender` from model instanc to document instance? the `instance_list` is `HTMLFile` object list, but you are sending `PageDocument` as sender.